python/connecting-ssl.py

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env python
 
 from couchbase.bucket import Bucket
+from couchbase.cluster import *
 
 # Note the `couchbases` in the scheme. This is required for SSL connections!
 cb = Bucket('couchbases://10.0.0.31/default?certpath=/tmp/couchbase-ssl-certificate.pem')
-print cb.server_nodes
+print(cb.server_nodes)
+
+cb_cluster = Cluster("http://10.0.0.31/")
+authenticator = CertAuthenticator(cert_path="/tmp/couchbase-ssl-certificate.pem", key_path="/tmp/couchbase-ssl-key.key", cluster_username="admin",
+                                  cluster_password="password")
```

Comment:
Could you point to certificates generated by the scripts from this repository? This should make it easier to execute for people.
https://github.com/couchbaselabs/devguide-examples/blob/18bccfece1087e1e8d7fac1b6e2fda2f1e6f2260/php/connecting-cert-auth.php#L3-L9

Also I would recommend using either loopback address (assuming default deployment of server on local machine), or drop some notes why this sample is using `10.0.0.31`.